src/3_branch.js

```diff
@@ -947,14 +933,13 @@ Branch.prototype['credits'] = wrap(callback_params.CALLBACK_ERR_DATA, function(d
 	this._api(resources.credits, { }, done);
 });
 
-
-if (CORDOVA_BUILD) { // jshint undef:false
 /**
  * @function Branch.creditHistory
  * @param {Object} data - _optional_ - options controlling the returned history.
  * @param {function(?Error,Object=)=} callback - _required_ - returns an array with credit history data.
  *
  * This call will retrieve the entire history of credits and redemptions from the individual user.
+ * // wtf is "direction"?
```

Comment:
Yeah I found it in the api docs. It is direction, so I clarified it in this documentation.